src/Shared/LayoutRenderers/AspNetSessionValueLayoutRenderer.cs

```diff
@@ -93,36 +89,33 @@ protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
             }
 
             var context = HttpContextAccessor.HttpContext;
-            var contextSession = context.TryGetSession();
-            if (contextSession == null)
-                return;
-
-#if !ASP_NET_CORE
-            var value = PropertyReader.GetValue(item, contextSession, (session,key) => session.Count > 0 ? session[key] : null, EvaluateAsNestedProperties);
-#else
-            //because session.get / session.getstring also creating log messages in some cases, this could lead to stackoverflow issues. 
-            //We remember on the context.Items that we are looking up a session value so we prevent stackoverflows
```

Comment:
Can the above comment be restored? So it appears just above `new RendererReEntrantManager` ?

Reply:
done